cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -671,16 +712,14 @@ impl ClientTrait for MockClient {
             .pop_front();
         match mock_resp {
             Some(Response::Resolve(resp)) => Ok(resp),
-            Some(Response::Search(_)) => {
-                panic!("found search response, expected resolve response");
-            },
             Some(Response::Error(err)) => Err(ResolveError::Resolve(
                 err.try_into()
                     .expect("couldn't convert mock error response"),
             )),
-            None => {
-                panic!("expected mock response, found nothing");
-            },
+            _ => panic!(
+                "expected resolve response, found {:?}",
+                print_type(&mock_resp.unwrap())
```

Comment:
suggestion: use `{:?}` or `{:#?}` only.
Either should include the variant and its likely easier to debug knowing the actual (incorrect) input.

Reply:
also `print_type` is _printing_ the type and returning `()`, this panic would likely return

```
"flox_rust_sdk::providers::catalog::Response"
thread 'providers::catalog::...' panicked at flox-rust-sdk/src/providers/catalog.rs
expected resolve response, found ()
``` 

which will actually omit the variant

Reply:
Fixed up, thanks!